apis/flinkcluster/v1beta1/flinkcluster_types.go

```diff
@@ -195,6 +195,9 @@ type JobManagerSpec struct {
 	// Currently `VPC, External` are only available for GKE.
 	AccessScope string `json:"accessScope,omitempty"`
 
+	// _(Optional)_ Define JobManager Service annotations for configuration.
+	ServiceAnnotations map[string]string `json:"ServiceAnnotations,omitempty"`
```

Comment:
Could you add `ServiceLabels` as well?

Reply:
Re this change, should I merge labels in this way:
```
var serviceLabels = mergeLabels(podLabels, jobManagerSpec.ServiceLabels, getRevisionHashLabels(&flinkCluster.Status.Revision))
```

Or should I exclusively define `var.serviceLabels` to `jobManagerSpec.ServiceLabels` ?

Reply:
> Or should I exclusively define var.serviceLabels to jobManagerSpec.ServiceLabels ?

Yeah, I think we should go with that and not inherit the pod labels

Reply:
We should still set the revision hash

Reply:
Done in 1bfa0f9
Had to remove few pre-defined lines from tests though